src/agent/multitype_queue/src/sqlitestorage.cpp

```diff
@@ -257,9 +294,9 @@ int SQLiteStorage::RemoveMultiple(int n, const std::string& tableName, const std
     }
     else
     {
-        constexpr std::string_view DELETE_MULTIPLE_QUERY {
-            "DELETE FROM {} WHERE module LIKE \"{}\" AND rowid IN (SELECT rowid FROM {} WHERE module LIKE \"{}\" ORDER "
-            "BY rowid ASC LIMIT ?);"};
+        constexpr std::string_view DELETE_MULTIPLE_QUERY {"DELETE FROM {} WHERE module_name LIKE \"{}\" AND rowid IN "
```

Comment:
Same here.

Reply:
In this first stage, because of the way events are being sent, it is not necessary, but it should be done in the future.